Content.Client/GameObjects/Components/Wires/WiresMenu.cs

```diff
@@ -147,14 +147,14 @@ public WiresMenu(WiresBoundUserInterface owner)
                 {
                     (_nameLabel = new Label
                     {
-                        Text = Loc.GetString("Wires"),
+                        Text = Loc.GetString("wires-wires"),
                         FontOverride = font,
                         FontColorOverride = StyleNano.NanoGold,
                         VerticalAlignment = VAlignment.Center,
                     }),
                     (_serialLabel = new Label
                     {
-                        Text = Loc.GetString("DEAD-BEEF"),
+                        Text = Loc.GetString("wires-dead-beef"),
```

Comment:
this is a tricky one, `DEAD-BEEF` is hexadecimal, it's not *really* in english at all.
I probably wouldn't translate this at all.